Applying lowpass and timescale causes the track to break.

## Description
<!-- Please describe the issue in details -->
Applying lowpass with timescale causes the track to break, with the following error `Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2`

Using them individually is not an issue, and combining lowpass/timescale with other filters is also not an issue.

## Version info
**Client used:** Modified version of https://github.com/Cog-Creators/Red-Lavalink

Spring logs - https://github.com/freyacodes/Lavalink/files/6570874/spring.log

```
        Version:        c31aecce8e7095eade87bb4d0775c7942487c6ff-SNAPSHOT
        Build:          1235
        Build time:     31.05.2021 11:59:03 UTC
        Branch          dev
        Commit:         c31aecc
        Commit time:    31.05.2021 11:56:03 UTC
        JVM:            11.0.11
        Lavaplayer      1.3.77
```


```
2021-05-31 15:25:04.228  INFO 26072 --- [  XNIO-1 I/O-14] lavalink.server.io.SocketServer          : {"op": "filters", "guildId": "133049272517001216", "volume": 1.0, "equalizer": [{"band": 0, "gain": -0.075}, {"band": 1, "gain": 0.125}, {"band": 2, "gain": 0.125}, {"band": 3, "gain": 0.0}, {"band": 4, "gain": 0.0}, {"band": 5, "gain": 0.0}, {"band": 6, "gain": 0.0}, {"band": 7, "gain": 0.0}, {"band": 8, "gain": 0.0}, {"band": 9, "gain": 0.0}, {"band": 10, "gain": 0.0}, {"band": 11, "gain": 0.0}, {"band": 12, "gain": 0.0}, {"band": 13, "gain": 0.0}, {"band": 14, "gain": 0.0}], "timescale": {"speed": 1.17, "pitch": 1.2, "rate": 1.0}, "lowPass": {"smoothing": 50.0}}
2021-05-31 15:25:04.447 ERROR 26072 --- [back-1-thread-1] c.s.d.l.t.p.LocalAudioTrackExecutor      : Error in playback of PIh2xe4jnpk

com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something went wrong when decoding the track.
        at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:44) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.executeProcessingLoop(LocalAudioTrackExecutor.java:293) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.executeProcessingLoop(LocalAudioTrackExecutor.java:249) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.process(MatroskaAudioTrack.java:38) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.track.DelegatedAudioTrack.processDelegate(DelegatedAudioTrack.java:25) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.processStatic(YoutubeAudioTrack.java:57) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.process(YoutubeAudioTrack.java:49) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:104) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:348) ~[lavaplayer-1.3.77.jar!/:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
        at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
        Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation:
  lavaplayer.version: 1.3.77
  os.arch: amd64
  os.name: Windows 10
  os.version: 10.0
  java.vendor: OpenLogic
  java.version: 11.0.11
  java.runtime.version: 11.0.11+9-adhoc..jdk11u
  java.vm.version: 11.0.11+9-adhoc..jdk11u
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2
        at com.github.natanbc.lavadsp.timescale.TimescalePcmAudioFilter.process(TimescalePcmAudioFilter.java:324) ~[lavadsp-0.7.7.jar!/:0.7.7]
        at com.github.natanbc.lavadsp.ConverterPcmAudioFilter.process(ConverterPcmAudioFilter.java:75) ~[lavadsp-0.7.7.jar!/:0.7.7]
        at com.sedmelluq.discord.lavaplayer.filter.converter.ToFloatAudioFilter.process(ToFloatAudioFilter.java:65) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.filter.UserProvidedAudioFilters.process(UserProvidedAudioFilters.java:76) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.filter.AudioPipeline.process(AudioPipeline.java:33) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.container.common.OpusPacketRouter.passDownstream(OpusPacketRouter.java:160) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.container.common.OpusPacketRouter.process(OpusPacketRouter.java:96) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaOpusTrackConsumer.consume(MatroskaOpusTrackConsumer.java:46) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.processFrameInBlock(MatroskaStreamingFile.java:371) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.parseClusterSimpleBlock(MatroskaStreamingFile.java:335) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.parseNextCluster(MatroskaStreamingFile.java:320) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.provideFrames(MatroskaStreamingFile.java:296) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.lambda$process$0(MatroskaAudioTrack.java:39) ~[lavaplayer-1.3.77.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.executeProcessingLoop(LocalAudioTrackExecutor.java:275) ~[lavaplayer-1.3.77.jar!/:na]
        ... 10 common frames omitted

```

--- a/LavalinkServer/src/main/java/lavalink/server/player/filters/filterConfigs.kt
+++ b/LavalinkServer/src/main/java/lavalink/server/player/filters/filterConfigs.kt
@@ -160,7 +160,7 @@ class LowPassConfig(
     private val smoothing: Float = 20.0f
 ) : FilterConfig() {
     override fun build(format: AudioDataFormat, output: FloatPcmAudioFilter): FloatPcmAudioFilter {
-        return LowPassPcmAudioFilter(output, format.sampleRate)
+        return LowPassPcmAudioFilter(output, format.channelCount)
             .setSmoothing(smoothing)
     }
 

